web/src/app/modules/views/side/modules/links-actions/components/links-actions.component.ts

```diff
@@ -28,12 +28,18 @@ export class LinksActions {
         return this.additionalInformationService.element;
     }
 
-    public get model(): IContainer {
-        return this.additionalInformationService.model;

```

Comment:
This changes the behavior. Now, _model is undefined instead of getting the model from the additionalInformationService